axelor-account/src/main/resources/domains/FixedAsset.xml

```diff
@@ -53,6 +85,36 @@
 		public static final String COMPUTATION_METHOD_LINEAR = "linear";
 		public static final String COMPUTATION_METHOD_DEGRESSIVE = "degressive";
 		
+		// DEPRECIATION PLAN SELECT
+		public static final String DEPRECIATION_PLAN_NONE = "0";
+		public static final String DEPRECIATION_PLAN_FISCAL = "1";
+		public static final String DEPRECIATION_PLAN_ECONOMIC = "2";
+		public static final String DEPRECIATION_PLAN_DEROGATION = "3";
```

Comment:
This should be integer, not string

Reply:
Since depreciationPlan is a multi-select, it need to be a string because it seems type integer is not managed by multi-select